src/Features/Core/Portable/Workspace/CompileTimeSolutionProvider.cs

```diff
@@ -41,7 +41,7 @@ public Factory()
         }
 
         private const string RazorEncConfigFileName = "RazorSourceGenerator.razorencconfig";
-        private const string RazorSourceGeneratorAssemblyName = "Microsoft.NET.Sdk.Razor.SourceGenerators";
+        private const string RazorSourceGeneratorAssemblyName = "Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators";
```

Comment:
This is just an intermediate assembly name while we verify things aren't broken. I'd say lets include both the intermediate and final assembly names here, then we can trip the intermediate when we're done with it.

Reply:
I'm not sure what you mean? How would I update uses of this constant if I added both names here?

I think when we are done with this intermediate assembly, we will redo this PR, renaming

```diff
-Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators
+Microsoft.CodeAnalysis.Razor.Compiler
```

Reply:
Just have multiple names.

Reply:
To spell out my concern more explicitly: coordinated insertions are pain. Anything we can do to avoid needing to coordinate insertions between multiple repos is good, and if we need to add a bit of extra code to handle "next state" and "next state + 1" at the same time, I think that's worth it.

Reply:
> Just have multiple constants.

Yes, but the current constant is used in `GetGeneratedDocumentPathWithoutExtension` (and maybe elsewhere). How should the second new constant be used?

Reply:
Think we should include the old assembly names, the intermediate and new assembly names. This will help support us thourgh the transition period. Can come back later and clean it up once we are past it.

Reply:
Talked with Jared offline, I now understand more what you mean, will look into it.